web/pgadmin/utils/__init__.py

```diff
@@ -984,3 +984,9 @@ def get_safe_post_logout_redirect():
         if url.startswith(item):
             return url
     return url_for('security.login')
+
+
+def check_extension_exists(conn, extension_name):
```

Comment:
If the function name is check_extension_exists then it should result should be either true or false.